modules/tinymce/src/core/test/ts/browser/DragDropOverridesTest.ts

```diff
@@ -117,4 +119,87 @@ describe('browser.tinymce.core.DragDropOverridesTest', () => {
     ]);
     await pAssertNotification('Dropped file type is not supported');
   });
+
+  it('TINY-8874: Dragging CEF element towards the bottom edge causes scrolling', async () => {
+    const editor = hook.editor();
+    await Waiter.pWait(200); // Wait a small amount of time to ensure the events have been bound
+    editor.setContent(`
+      <p contenteditable="false" style="height: 200px; background-color: black; color: white">Draggable CEF</p>
+      <p id="separator" style="height: 100px"></p>
+      <p id="hidden" style="height: 300px"></p>
+      <p>CEF can get dragged after this one</p>
+    `);
+    const target = UiFinder.findIn(TinyDom.body(editor), 'p:contains("Draggable CEF")').getOrDie();
+    const dest = UiFinder.findIn(TinyDom.body(editor), '#separator').getOrDie();
+    const initialScrollY = editor.getBody().ownerDocument.defaultView.scrollY;
+    Mouse.mouseDown(target);
+    Mouse.mouseMoveTo(dest, 0, 98); // Move the mouse close to the bottom edge of the editor to trigger scrolling
+    await Waiter.pWait(1000); // Wait a small amount of time to ensure the scrolling happens
+    Mouse.mouseUp(target);
+    assert.isTrue(editor.getBody().ownerDocument.defaultView.scrollY > initialScrollY); // Make sure scrolling happened
```

Comment:
This should have been using `isAbove` like you did in the test below. Reason being you can't see the values when it fails so the error message information is very unhelpful if you do this type of thing.